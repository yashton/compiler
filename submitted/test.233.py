s = "This is the #comment\
 string"
