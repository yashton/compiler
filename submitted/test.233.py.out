(ID "s")
(PUNCT "=")
(ERROR "Illegal character '"This is the #comment\
 string"
' on line 1)
