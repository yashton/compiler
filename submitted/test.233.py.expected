(ID "s")
(PUNCT "=")
(LIT "This is the #comment string")
(NEWLINE)
(ENDMARKER)
